nni/compression/experiment/experiment.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from __future__ import annotations
+import logging
+from pathlib import Path
+from typing import Any, Callable, List
+
+import torch
+from torch.nn import Module
+from torch.optim import Optimizer
+
+from nni.compression.experiment.config import generate_compression_search_space
+from nni.experiment import Experiment
+from .config import CompressionExperimentConfig, CompressionVessel
+
+_logger = logging.getLogger('nni.experiment')
+
+
+class CompressionExperiment(Experiment):
+    def __init__(self, config_or_platform: CompressionExperimentConfig | str | List[str] | None,
```

Comment:
I don't really understand. You already had a `CompressionExperimentConfig` which contains everything. Why do you need model, fine tuner, evaluator and etc. as extra arguments?

Reply:
`CompressionExperimentConfig` don't handle model, finetuner..., maybe need more discussion about config...